src/knx/ip_data_link_layer.cpp

```diff
@@ -67,6 +76,10 @@ void IpDataLinkLayer::loop()
             KnxIpSearchResponse searchResponse(_ipParameters, _deviceObject);
 
             auto hpai = searchRequest.hpai();
+#ifdef KNX_ACTIVITYCALLBACK
+    if(_dllcb)
```

Comment:
Indent looks of. Is it a spaces vs. tabs issue?

Reply:
fixed